specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,2712 @@
+          "items": {
+            "$ref": "#/definitions/Module"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "geoReplication": {
+          "type": "object",
+          "description": "Optional set of properties to configure geo replication for this database.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "properties": {
+            "groupNickname": {
+              "type": "string",
+              "description": "Name for the group of linked database resources",
+              "x-ms-mutability": [
+                "read",
+                "create"
+              ]
+            },
+            "linkedDatabases": {
+              "type": "array",
+              "description": "List of database resources to link with this database",
+              "items": {
+                "$ref": "#/definitions/LinkedDatabase"
+              }
+            }
+          }
+        },
+        "redisVersion": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Version of Redis the database is running on, e.g. '6.0'"
+        },
+        "deferUpgrade": {
+          "type": "string",
+          "description": "Option to defer upgrade when newest version is released - default is NotDeferred. Learn more: https://aka.ms/redisversionupgrade",
+          "enum": [
+            "Deferred",
+            "NotDeferred"
+          ],
+          "x-ms-enum": {
+            "name": "DeferUpgradeSetting",
+            "modelAsString": true
+          }
+        },
+        "accessKeysAuthentication": {
+          "type": "string",
+          "description": "Access database using keys - default is enabled. This property can be Enabled/Disabled to allow or deny access with the current access keys. Can be updated even after database is created.",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "x-ms-enum": {
+            "name": "accessKeysAuthentication",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "DatabaseList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Database"
+          },
+          "description": "List of databases"
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The URI to fetch the next page of results."
+        }
+      },
+      "description": "The response of a list-all operation."
+    },
+    "AccessPolicyAssignmentProperties": {
```

Comment:
>AccessPolicyAssignmentProperties [](http://example.com/codeflow?start=5&length=32)

Is there an API for creating the access policies that this references also, or not, for this version?

Reply:
for this version only default is supported, APIs to manage different AccessPolicies will be created later.

Reply:
only default is supported for now.